DXMainClient/Domain/Multiplayer/MultiplayerColor.cs

```diff
@@ -17,6 +18,10 @@ public class MultiplayerColor
 
         private static List<MultiplayerColor> colorList;
 
+        private static string randomColorLabel;
+
+        private static readonly string RandomColorDefaultLabel = "Random".L10N("UI:Main:RandomSide");
```

Comment:
```suggestion
        private static readonly string RandomColorDefaultLabel = "Random".L10N("UI:Main:RandomColor");
```

And also considering making this a readonly property to prevent L10N being called before loading the translation table (although it should works without being a property as long as the class is initialized after L10N initialized but it is not straightforward enough).

Reply:
I'm not sure I understand. It is currently `readonly`. Are you saying it _shouldn't_ be readonly?

Reply:
Good catch on the incorrect string.

Reply:
> I'm not sure I understand. It is currently `readonly`. Are you saying it _shouldn't_ be readonly?

No. I mean changing `=` to `=>`

Reply:
Ahhhhh, I got it. Will do.

Reply:
Done.